src/applications/coronavirus-chatbot/createCoronavirusChatbot.js

```diff
@@ -10,23 +10,30 @@ export default (_store, widgetType) => {
     return;
   }
 
-  import(/* webpackChunkName: "chatbot" */ './index')
-    .then(module => {
-      const initializeChatbot = module.default;
-      initializeChatbot(root);
-    })
-    // eslint-disable-next-line no-unused-vars
-    .then(res => {
-      recordEvent({
-        event: `${GA_PREFIX}-load-successful`,
-        'error-key': undefined,
-      });
-    })
-    // eslint-disable-next-line no-unused-vars
-    .catch(error => {
-      recordEvent({
-        event: `${GA_PREFIX}-load-failure`,
-        'error-key': undefined,
+  import(/* webpackChunkName: "chatbot" */ './index').then(module => {
+    const initializeChatbot = module.default;
+    initializeChatbot()
+      .then(webchatOptions => {
+        recordEvent({
+          event: `${GA_PREFIX}-connection-successful`,
+          'error-key': undefined,
+        });
+        recordEvent({
+          event: `${GA_PREFIX}-load-successful`,
+          'error-key': undefined,
+        });
+
+        window.WebChat.renderWebChat(webchatOptions, root);
+      })
+      .catch(() => {
+        recordEvent({
+          event: `${GA_PREFIX}-connection-failure`,
+          'error-key': 'XX_failed_to_start_chat',
+        });
+        recordEvent({
+          event: `${GA_PREFIX}-load-failure`,
+          'error-key': undefined,
+        });
```

Comment:
Optionally, you could switch to async/await
```suggestion
  import(/* webpackChunkName: "chatbot" */ './index').then(async module => {
    const initializeChatbot = module.default;
    try {
        const webchatOptions = await initializeChatbot();
        recordEvent({
          event: `${GA_PREFIX}-connection-successful`,
          'error-key': undefined,
        });
        recordEvent({
          event: `${GA_PREFIX}-load-successful`,
          'error-key': undefined,
        });
        window.WebChat.renderWebChat(webchatOptions, root);
    } catch(err) {
      recordEvent({
          event: `${GA_PREFIX}-connection-failure`,
          'error-key': 'XX_failed_to_start_chat',
        });
        recordEvent({
          event: `${GA_PREFIX}-load-failure`,
          'error-key': undefined,
        });
    }
```